packages-content-model/roosterjs-content-model-plugins/lib/autoFormat/keyboardListTrigger.ts

```diff
@@ -0,0 +1,59 @@
+import { createListItem, createListLevel } from 'roosterjs-content-model-dom';
+import { getListTypeStyle } from './utils/getListTypeStyle';
+import {
+    getOperationalBlocks,
+    isBlockGroupOfType,
+    updateListMetadata,
+} from 'roosterjs-content-model-core';
+import type { IStandaloneEditor } from 'roosterjs-content-model-types';
+
+/**
+ * @internal
+ */
+export default function keyboardListTrigger(
+    editor: IStandaloneEditor,
+    shouldSearchForBullet: boolean = true,
```

Comment:
Can we reuse the options object here?

Reply:
We can reuse, but options might have properties not related to auto list. Also we are using this properties before calling this function, because if both autoBullet and autoNumbering are false, we do not need to call keyboardListTrigger.

Reply:
ok

Reply:
So as an internal function, we don't expect it to be called from other place, so no need to have default values of the parameters.